### ISSUE TITLE
Incorrect namespace for ReindexProduct observer

### ISSUE THREAD

**Issue by baldwinagency-pieter** [mollie/magento2#249]
Hi guys

This requires composer version 1.10.0 or higher (but lower than version 2).

After installing mollie/magento2 version 1.12.2, and running `composer dump-autoload -o`, it will show this warning:
```
Deprecation Notice: Class Mollie\Payment\Observer\ReindexProduct located in ./vendor/mollie/magento2/Observer/SalesOrderUnCancelAfterCommit/ReindexProduct.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///some/path/to/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:201
```

It looks like the namespace of that file should be `Mollie\Payment\Observer\SalesOrderUnCancelAfterCommit` instead of `Mollie\Payment\Observer`

Thanks!

**Comment by baldwinagency-pieter** [comment]
Somebody should probably also figure out if that class is actually still being used, because I can't find references in the module to it ...

**Comment by Frank-Magmodules** [comment]
@baldwinagency-pieter, Thank you for opening this issue.
We removed ReindexProduct.php as it's obsolete now. This fix is currently being tested so that it can be included in the release early next week, we will update this issue once the release is there.

Frank

**Comment by Frank-Magmodules** [comment]
Hi @baldwinagency-pieter, thanks again for the detailed issue report. The issue has been resolved in the [v1.13.0](https://github.com/mollie/magento2/releases/tag/v1.13.0) release. We are closing this issue now but please feel free to reopen the issue if this still occurs.

**Comment by hostep** [comment]
Thanks @Frank-Magmodules, I can confirm that the issue got fixed in v1.13.0!
